xml/art_sle_ha_pmremote.xml

```diff
@@ -274,50 +301,213 @@
+        The key for the &pmremote; service is different from the cluster
+        authentication key that you create in the &yast; cluster module.</para>
+      </step>
+      <step>
+       <para>In <filename>/etc/csync2/csync2.cfg</filename>, add the
+        following line:</para>
+       <screen>include /etc/pacemaker/authkey</screen>
+      </step>
+      <step>
+       <para>
+        List all cluster and remote nodes in the &csync; configuration file.
+       </para>
+      </step>
+   </procedure>
+  </sect2>
+  <sect2 xml:id="sec.ha.pmremote.config.uc1.kvm.guest">
+   <title>Configuring the KVM Guests</title>
+   <para>
+    The following procedure creates a KVM guest node <systemitem
+      class="domainname">&node4;</systemitem> on your physical remote host
+    <systemitem class="domainname">&node3;</systemitem>:
+   </para>
+   <procedure>
     <step>
-     <para>On two physical hosts, set up a basic two-node cluster as
-      described in <citetitle>&instquick;</citetitle>.
-     <!--<xref linkend="art.ha.install.quick"/>--></para>
+     <para>Create a KVM guest on <systemitem class="domainname"
+      >&node3;</systemitem>. For details refer to the
+      <citetitle>&virtual;</citetitle>, chapter <citetitle>Guest Installation</citetitle>
+      available from <link xlink:href="https://www.suse.com/doc"/>.
+     </para>
     </step>
     <step>
-     <para>On &node1;, proceed as follows:</para>
+     <para>On <systemitem class="domainname"
+      >&node4;</systemitem>, proceed as follows:</para>
      <substeps>
-      <!--
       <step>
        <para>In the firewall settings, open the TCP port 3121 for
         &pmremote;.</para>
       </step>
-      -->
+      <step>
+       <para>Install the <package>pacemaker-remote</package> and
+         <package>crmsh</package> packages: </para>
+       <screen>&prompt.root;<command>zypper</command> in pacemaker-remote crmsh</screen>
+      </step>
+      <step>
+       <para>Enable and start the &pmremote; service on <systemitem
+         class="domainname">&node3;</systemitem>:</para>
+       <screen>&prompt.root;<command>systemctl</command> enable pacemaker_remote
+&prompt.root;<command>systemctl</command> start pacemaker_remote</screen>
+      </step>
+      <step>
+       <para>Create the <filename>/etc/pacemaker</filename> directory and
+        change its group:</para>
+       <screen>&prompt.root;<command>mkdir</command> -p --mode=0750 /etc/pacemaker
+&prompt.root;<command>chgrp</command> haclient /etc/pacemaker</screen>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>On <systemitem class="domainname">&node3;</systemitem>,
+      run <command>csync2</command> to copy the authentication key from
+      your physical host to your guest node:</para>
+     <screen>&prompt.root;<command>csync</command> -xv</screen>
+    </step>
+    <step>
+     <para>
+      To verify the host connection to the guest, run <command>ssh</command>
```

Comment:
@gao-yan, shouldn't it be: s/guest/guest node/ or alternatively s/guest/virtual remote node/ ?